ovos_utils/time.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+from dateutil.tz import gettz, tzlocal
+from typing import Any
+
+# used to calculate timespans
+DAYS_IN_1_YEAR = 365.2425
+DAYS_IN_1_MONTH = 30.42
+
+
+def get_tz() -> Any:
```

Comment:
im thinking this name is ambiguous for users that import it, perhaps keep `get_config_tz` or `get_default_tz` ?